### DISCUSSION TITLE
Is it possible to delete an application from recents?

### DISCUSSION THREAD

**Discussion by advilm** [Ulauncher/Ulauncher#960] [Q&A] [answered]
Hello, I started using Ulauncher recently and I was wondering if there was a way to essentially reset an app. This would make it as if the app was opened 0 times. I found `/home/<username>/.local/share/ulauncher/app_stat_v2.db` which seems to have what I'm looking for. However, it's not a plaintext file. Is there a simple way to edit it?

**Comment by friday** [comment, marked as answer]
Hi!

Your only options for Ulauncher v5 are to delete it or edit with python. The upcoming Ulauncher v6 release has replaced these settings files with JSON, so it will be easier to change.

**Reply by advilm** [reply]
Alright, I'll be looking forward to that release!
